apps/native/app/src/screens/wallet/wallet.tsx

```diff
@@ -110,6 +111,7 @@ export const WalletScreen: NavigationFunctionComponent = ({ componentId }) => {
           GenericLicenseType.FirearmLicense,
           showDisability ? GenericLicenseType.DisabilityLicense : null,
           showPCard ? GenericLicenseType.PCard : null,
+          showEhic ? GenericLicenseType.Ehic : null,
         ].filter(Boolean) as GenericLicenseType[],
```

Comment:
~I wrote a is Defined function here back in the day

https://github.com/island-is/island.is/blob/main/libs/shared/utils/src/lib/isDefined.ts

it is especially handy when you are filtering null or undefined values from list. That way you skip the TS casting
Usage is basically 

```
list.filter(isDefined)
```